src/Select/Select.tsx

```diff
@@ -126,6 +141,7 @@ const extractValue = (options, isMulti) => {
   }
 };
 
+// petror@nulogy.dev: TODO: update package "react-windowed-select" to latest version
```

Comment:
Did you want to do this in this branch?

Reply:
@haideralsh , I am not sure, maybe make sense to move to another task. Just wanted to put some reminder. Because we have very old version, event without TS support.
The latest one has TS support, but requires some updates from our side too.

Reply:
It is not required for this task, but would be good to make this update later. I can create additional task on the board.

Reply:
Ok in that case let's remove the comment and add this as a Jira task, since if left in the code we will not remember it.